packages/manager/modules/storages/src/netapp/share/shares.template.html

```diff
@@ -0,0 +1,60 @@
+<div class="col-md-12">
+    <!-- SHARE -->
+    <h4 class="oui-tile__title oui-heading_4" data-translate="share_title"></h4>
```

Comment:
`oui-tile_title` class is not required, and won't be useful
```suggestion
    <h4 class="oui-heading_4" data-translate="share_title"></h4>
```